lib/EnsEMBL/REST/Model/ga4gh/ga4gh_utils.pm

```diff
@@ -0,0 +1,119 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+# shared configuration etc
+package EnsEMBL::REST::Model::ga4gh::ga4gh_utils;
```

Comment:
Can this be taken out of the Catalyst Model area and put somewhere like the ensembl (perhaps ga4gh) models? It doesn't look like this code should need to know anything about the server and can be freed from it. Then its invocation does not need the context elsewhere which will clean up some verbose code.